src/main/java/com/truelayer/java/merchantaccounts/entities/ListTransactionsQuery.java

```diff
@@ -0,0 +1,18 @@
+package com.truelayer.java.merchantaccounts.entities;
+
+import com.truelayer.java.merchantaccounts.entities.transactions.TransactionTypeQuery;
+import java.time.ZonedDateTime;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.experimental.Accessors;
+
+@Builder
+@Getter
+@Accessors(fluent = true)
+public class ListTransactionsQuery {
+    private ZonedDateTime from;
```

Comment:
The use of ZonedDateTime is something I'd really get some feedback

Reply:
Yup, I'd ask @azanin to confirm on this but my understanding is that the [`Get Transactions` endpoint](https://docs.truelayer.com/reference/get_merchant-accounts-id-transactions) accepts any datetime in ISO 8601. Therefore, so long as the datetime field (`from`/`to` in this case) ships a Time zone information, it's fine. Whatever the timezone the end user will send it will be transformed in UTC and then used to query our stores

Reply:
I confirm what @dili91 said